src/css/components/table.css

```diff
@@ -0,0 +1,140 @@
+/**
+ * @file
+ * Styles for Olivero's tables.
+ */
+ 
+table {
+  width: fit-content;
+  max-width: 100%;
+  border-collapse: collapse;
+  display: block;
+  margin: var(--sp2) 0;
+  border-spacing: 0;
+  border: 0;
+  font-family: var(--font-sans);
+  line-height: var(--sp1-5);
+  font-size: 16px;
+  color: var(--color--gray-10);
+  overflow-x: auto;
+  -webkit-overflow-scrolling: touch;
+
+  /* Hide scrollbar */
+  scrollbar-width: none; /* Firefox */
+  -ms-overflow-style: none;  /* Internet Explorer 10+ */
+
+  &:-webkit-scrollbar { /* WebKit */
+    width: 0;
+    height: 0;
+  }
+  
+  background:
+        /* Left start and right start 'inside' container colors (they overlap the shadows) */
+				linear-gradient(90deg, white 0%, rgba(255,255,255,0)),
+        linear-gradient(-90deg, white 0%, rgba(255,255,255,0)) 100% 0,
+        /* Left and right scroll shadows */
+				radial-gradient(
+					farthest-side at 0% 50%,
+					rgba(0,0,0,0.2),
+					rgba(0,0,0,0)
+				),
+				radial-gradient(
+					farthest-side at 100% 50%,
+					rgba(0,0,0,0.2),
+					rgba(0,0,0,0)
+				) 100% 0%;
+			background-repeat: no-repeat;
+			background-color: white;
+			background-size: 100px 100%, 100px 100%, 14px 100%, 14px 100%;
+			background-attachment: local, local, scroll, scroll;
+
+  thead {
+    tr {
```

Comment:
Do we need this thead and tr for specificity? I'd prefer to keep things less specific

Reply:
I think we do, as the border-bottom color of the `thead` is `color--blue-50` instead of `color--gray-40`.